src/layouts/SiteDashboard/SiteDashboard.tsx

```diff
@@ -58,6 +67,16 @@ export const SiteDashboard = (): JSX.Element => {
     isLoading: isCollaboratorsStatisticsLoading,
   } = useGetCollaboratorsStatistics(siteName)
 
+  const savedAt = getDateTimeFromUnixTime(siteInfo?.savedAt || 0)
+  const publishedAt = getDateTimeFromUnixTime(siteInfo?.publishedAt || 0)
+
+  useEffect(() => {
+    // GitHub users should not be able to access this page
+    if (userId !== "Unknown user" && !!userId) {
+      setRedirectToPage(`/sites/${siteName}/workspace`)
+    }
+  })
```

Comment:
I'm wondering how many places we need to do this redirection - is it just here and collaborators? If there are more i'm wondering if it's better to have a different type of route which automatically restricts access and redirects instead, but regardless i think this can be settled in a future pr if necessary

Reply:
Not collaborators but will be needed for review requests. I think this can be handled in the future together with the review requests side of things? @seaerchin